src/proto-root/utils.js

```diff
@@ -2,32 +2,36 @@
 
 const ProtoBuf = require('protobufjs');
 
+const UNDEFINED = 0;
+const STRING = 1;
+const JPEG = 2;
+const DATE = 3;
+const PNG = 4;
+
 module.exports.toCamelCase = str => ProtoBuf.Util.toCamelCase(str);
 
 module.exports.convertValueBasedOnContentType = (value, contentType) => {
-  const UNDEFINED = 0;
-  const STRING = 1;
-  const JPEG = 2;
-  const DATE = 3;
-  const PNG = 4;
-
   switch (contentType) {
     // UNDEFINED should not be seen, and is used as an error placeholder
     case UNDEFINED:
-      throw new Error('wrong content type');
-    // STRING means the value is UTF-8 encoded text.
-    case STRING:
+      throw new Error('Wrong content type');
+    case STRING: // STRING means the value is UTF-8 encoded text.
+    case DATE: // Date as string in RFC3339 format (YYYY-MM-DD).
       return value.toUTF8();
+    default:
+      return value;
+  }
+};
+
+module.exports.imageUriBasedOnContentType = (value, contentType) => {
+  switch (contentType) {
     // JPEG indicates a standard .jpeg image.
     case JPEG:
       return `data:image/jpeg;base64,${value.toBase64()}`;
-    // Date as string in RFC3339 format (YYYY-MM-DD).
-    case DATE:
-      return value.toUTF8();
     // PNG indicates a standard .png image.
     case PNG:
       return `data:image/png;base64,${value.toBase64()}`;
     default:
-      return value;
+      return `data:image/jpeg;base64,${value.toBase64()}`;
```

Comment:
Why do we treat this as a JPEG?

Reply:
No reason, just assuming the default image type is JPEG. the default branch shouldn't exist, but I have to return something because of the arrow function. What would you prefer? `return value`? `return value.toBase64()`.

Reply:
Yes I guess I was thinking it should never get to this point if it is a JPEG, I think `value` would be best, but `value.toBase64` also ok